funnel/templates/logout_browser_data.html.jinja2

```diff
@@ -0,0 +1,17 @@
+{% extends "layout.html.jinja2" %}
+{% block title %}{% trans %}Logout{% endtrans %}{% endblock %}
+
```

Comment:
This needs a bit of content explaining that the logout is in progress. We have a similar case in `meta_refresh.html.jinja2`.

Reply:
This helps, but I was referring to the body. Users will see a blank page for a second or so. We might as well show "Logging out…" in the middle of the screen for that second.